P5/Source/Specs/origDate.xml

```diff
@@ -25,7 +25,8 @@ $Id$
   <desc versionDate="2007-01-21" xml:lang="it">contiene una data espressa in una forma qualsiasi utilizzata per indicare la data di origine di un manoscritto o di una sua parte</desc>
   <classes>
     <memberOf key="att.global"/>
-    <memberOf key="att.datable"/>
+    <memberOf key="att.datable" />
+    <!--<memberOf key="att.calendarSystem"/>--> <!-- Uncomment this after @calendar is deprecated on att.datable -->
```

Comment:
I wonder if this is necessary. Can we uncomment it now on dev branch or make the new class available before the deprecation period ends?